### ISSUE TITLE
Setup deployment 

### ISSUE THREAD

**Issue by Siegrift** [api3dao/airseeker-v2#7]
We should be able to run Airseeker locally and have it deployed on AWS. This task involves choosing a deployment method (for example, serverless framework or AWS tools or other) and setting it up.

This task to be done after #12.

**Comment by bbenligiray** [comment]
This will just be a container, the cloud deployment is out of scope

**Comment by aquarat** [comment]
I'm just confirming, if cloud deployment is out of scope+this is a container, then this task is a duplicate of https://github.com/api3dao/airseeker-v2/issues/12 ?

**Comment by bbenligiray** [comment]
Looks like it
